README.md

```diff
@@ -449,6 +467,32 @@ export default MyComponent = () => {
 
 ## Migrating between versions
 
+### Migrating from 4.x to 5.x
+
+From version 5.0, to remove coupling with `react-router`, you need to pass `navigate` prop to `Security`.
+For `react-router`:
+```jsx
+import { useHistory } from 'react-router-dom';
+
+export default App = () => {
+  const history = useHistory();
+    return (
+      <Router>
+        <Security oktaAuth={oktaAuth} navigate={history.push}>
+          // children component
+        </Security>
+      </Router>
+    );
+};
+```
+Also to use `SecureRoute`:
```

Comment:
Showing with code is always good, but we should also provide text to describe the intent (here, it appears to be that you're passing useOktaAuth)